Tools/TestWebKitAPI/Tests/WebKitLegacy/mac/URLExtras.mm

```diff
@@ -25,6 +25,7 @@
 
 #import "config.h"
 
+#import "PlatformUtilities.h"
 #import "Test.h"
```

Comment:
why is this file in WebKitLegacy?

Reply:
That's probably because the code that it exercises (`-_webkit_suggestedFilenameWithMIMEType`) lives in WebKitLegacy.

Reply:
Quite!